sdk/storage/storage-file-datalake/test/perfstress/track-2/upload.spec.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+import { StorageDFSTest } from "./storageTest.spec";
+
+interface StorageFileShareUploadTestOptions {
+  size: number;
+}
+
+export class StorageDFSUploadTest extends StorageDFSTest<StorageFileShareUploadTestOptions> {
+  public options: PerfStressOptionDictionary<StorageFileShareUploadTestOptions> = {
+    size: {
+      required: true,
+      description: "Size in bytes",
+      shortName: "sz",
+      longName: "size",
+      defaultValue: 1024
+    }
+  };
+
+  async runAsync(): Promise<void> {
+    await this.directoryClient
```

Comment:
Each test instance should upload to the same file repeatedly, so the file client should be created once per instance and reused.  The buffer should also be reused.